manoseimas/lobbyists/views/lobbyists.py

```diff
@@ -0,0 +1,30 @@
+# -*-coding: utf-8 -*-
+
+from django.shortcuts import render
+from manoseimas.lobbyists.models import Lobbyist
+
+def lobbyist_list(request):
+    """A placeholder."""
+    lobbyists = Lobbyist.objects.all()
+    return render(request, 'lobbyist_list.jade', {lobbyists: lobbyists})
```

Comment:
I think you want `{'lobbyists': lobbyists}` or `dict(lobbyists=lobbyists)` here.